internal/pkg/mount/v2/mount.go

```diff
@@ -298,6 +313,20 @@ func (p *Point) Move(newTarget string) error {
 	return unix.Mount(p.target, newTarget, "", unix.MS_MOVE, "")
 }
 
+// RemountReadOnly remounts the mount point as read-only.
+func (p *Point) RemountReadOnly() error {
+	p.flags |= unix.MS_RDONLY
+
+	return p.remount()
+}
+
+// RemountReadWrite remounts the mount point as read-write.
+func (p *Point) RemountReadWrite() error {
+	p.flags &^= unix.MS_RDONLY
```

Comment:
Is this equivalent to `&= ~unix.MS_RDONLY`?

Reply:
I think that should be exactly it.